src/Microsoft.OData.Edm/Csdl/CsdlJsonWriter.cs

```diff
@@ -49,13 +50,30 @@ protected override void WriteCsdl()
             WriteReferenceElements();
 
             // It also MAY contain members for schemas.
-            WriteSchemata();
+            WriteSchema();

```

Comment:
Why this change?

Reply:
@habbes This was suggested in the previous comment to be renamed to `WriteSchema`

Reply:
See my comments here: https://github.com/OData/odata.net/pull/3006#discussion_r1675475503

Reply:
@habbes @ElizabethOkerio 

Lets rename these private methods to `WriteSchemas` and `WriteSchemasAsync`

Reply:
This has been done.